src/main/java/org/jenkinsci/plugins/mesos/MesosPodRecordRepository.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.plugins.mesos;
+
+import akka.Done;
+import com.mesosphere.usi.repository.PodRecordRepository;
+import java.util.concurrent.CompletableFuture;
+import scala.collection.immutable.HashMap;
+import scala.collection.immutable.Map;
+import scala.compat.java8.FutureConverters;
+import scala.concurrent.Future;
+
+public class MesosPodRecordRepository implements PodRecordRepository {
+  @Override
+  public Future<Done> delete(Object record) {
+    CompletableFuture<Done> javaFuture = CompletableFuture.supplyAsync(() -> Done.done());
```

Comment:
Is `supplyAsync` required ? Can't we do this : 
```java
return FutureConverters.toScala(CompletableFuture.completedFuture(Done.done()));
```

Reply:
there is no completedFuture method on the java completable future